ServerWhitelist.yml

```diff
@@ -1947,19 +1947,15 @@ AllowedPrefixes:
     - https://cdn.discordapp.com/attachments/1063514401375780874/1198357469903204383/SexLab_Framework_PPLUS_-_V2.3.2.1_-_1.5.97.zip?ex=65da4bdd&is=65c7d6dd&hm=3121fe0ab2594a7a661e07a0d19c0dafe7f72813946e2be02479e04977e51a9b& # Sexlab + 2.3.2.1 / SE 1.5.9.7 ( Inline Permission from Scrab )
     - http://www.mediafire.com/file/o5yacfkx21p3b8i/Sexlab_DLL_16640_Replacer.rar # Sexlab Framework 16640 Working DLL ( Inline Permission from Scrab )
```

Comment:
Permission from Scrab, not sure if Screenshot needet.

Reply:
https://github.com/wabbajack-tools/opt-out-lists/pull/816/files#r1501290676

Reply:
Screenshot as Attachment in Zip File.